docs/docs/configuration/hardware_acceleration.md

```diff
@@ -242,3 +242,69 @@ processes:
 If you do not see these processes, check the `docker logs` for the container and look for decoding errors.
 
 These instructions were originally based on the [Jellyfin documentation](https://jellyfin.org/docs/general/administration/hardware-acceleration.html#nvidia-hardware-acceleration-on-docker-linux).
+
+### NVIDIA Jetson (Nano, NX, AGX)
```

Comment:
Can you clarify in the docs if this works with the original Nano, Xavier, Orin, etc.?

Reply:
Updated